lunar/distribution.yaml

```diff
@@ -144,16 +144,19 @@ repositories:
       packages:
       - astuff_sensor_msgs
       - delphi_esr_msgs
+      - delphi_mrr_msgs
       - delphi_srr_msgs
       - ibeo_msgs
       - kartech_linear_actuator_msgs
       - mobileye_560_660_msgs
       - neobotix_usboard_msgs
       - pacmod_msgs
+      - perception_msgs
```

Comment:
There is already a perception stack in ROS so this name will be misleading for users.
This package name doesn't allow to know what it contains and how it differs from general perception messages.
You may find useful messages in `vision_msgs` and `object_recognition_msgs` that may allow you to not define another set of messages for this.

Reply:
@mikaelarguedas - Thanks for the suggestions. I've looked through those messages briefly and they do not contain enough high-level fields (specifically the object's detected/estimated velocity and acceleration) to describe the objects for use in sensor fusion. However, we have a couple of other messages in there that are specific to use in autonomous vehicles. I'll see what we can do about renaming this to be more specific and re-submit the release.

Reply:
@mikaelarguedas - Any issues with `derived_object_msgs`?

Reply:
@mikaelarguedas - We've had some discussion internally. Here is what we came up with (also covers changes from #18765 :

- Change `perception_msgs` to `derived_object_msgs`.
- Change `platform_automation_msgs` to `automotive_autonomy_msgs`

Inside `automotive_autonomy_msgs`:

- Create `automotive_platform_msgs` and `automotive_navigation_msgs`
- Redistribute existing messages in `platform_comm_msgs` and `module_comm_msgs` to the above two and move a couple into `derived_object_msgs`.

Please let us know if you have any issues with these more-specific names. If not, we will reorganize and re-release under these names.

Reply:
thanks @JWhitleyAStuff for iterating on this.

Discussing this with @tfoote, the new package names are much clearer, thanks!

> Please let us know if you have any issues with these more-specific names. If not, we will reorganize and re-release under these names.

Sounds good to us :+1:, feel free to move forward with the reorganization and new release